openstack/networking/v2/extensions/vpnaas/ikepolicies/requests.go

```diff
@@ -105,3 +105,42 @@ func Create(c *gophercloud.ServiceClient, opts CreateOptsBuilder) (r CreateResul
 	_, r.Err = c.Post(rootURL(c), b, &r.Body, nil)
 	return
 }
+
+// UpdateOptsBuilder allows extensions to add additional parameters to the
+// Update request.
+type UpdateOptsBuilder interface {
+	ToPolicyUpdateMap() (map[string]interface{}, error)
+}
+
+type LifetimeUpdateOpts struct {
+	Units Unit `json:"units,omitempty"`
+	Value int  `json:"value,omitempty"`
+}
+
+// UpdateOpts contains the values used when updating an IKE policy
+type UpdateOpts struct {
```

Comment:
It looks like `Phase1Negotiation` can be updated: https://github.com/openstack/neutron-vpnaas/blob/master/neutron_vpnaas/extensions/vpnaas.py#L418